### PR TITLE
package(cmf-react-router)

### PR DESCRIPTION
**Pull request by romainseb**
**What is the problem this PR is trying to solve?**
Bring a cmf router for react-router v5
**What is the chosen solution to this problem?**

**Please check if the PR fulfills these requirements**

- [x] The PR have used `yarn changeset` to a request a release from the CI if wanted.
- [x] The PR commit message follows our [guidelines](https://github.com/talend/tools/blob/master/tools-root-github/CONTRIBUTING.md)
- [ ] Tests for the changes have been added (for bug fixes / features) And [non reg](./screenshots.md) done before need review
- [ ] Docs have been added / updated (for bug fixes / features)
- [ ] Related design / discussions / pages (not in jira), if any, are all linked or available in the PR

**[ ] This PR introduces a breaking change**

### CODE REVIEW

File: packages/cmf-react-router/README.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
+# @talend/cmf-react-router
+
+This module is like cmf-router but for React router v5 (and more recent) where @talend/cmf-router is tied up with react-router v3.
```
Comment (jsomsanith-tlnd) [inline packages/cmf-react-router/README.md:3]: I think you should write a real doc and not rely on a package that will be removed when everyone migrates to this one

File: packages/cmf-react-router/README.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+				onAddDatastore: call(
+					redirectOnEvent,
+					AddDatasetForm.events.DATASET_CREATION_PAUSED_AND_ADD_DATASTORE,
+					'/datasets/add/connection/add',
+				),
+				onSubmit: call(redirectOnSuccessEvent, AddDatasetForm.events.DATASET_CREATION_DONE),
+			});
+		},
+	}],
+	routerFunctions: {},
```
Comment (jsomsanith-tlnd) [inline packages/cmf-react-router/README.md:31]: `routerFunctions ` are not supported anymore here

File: .changeset/ci-dependencies.md [hunk 3]
```diff
@@ -1,6 +1,7 @@
 ---
 '@talend/react-cmf-router': patch
 '@talend/react-cmf': patch
+'@talend/cmf-react-router': patch
```
[the remark this answers is not in the document]
Comment (romainseb) [reply .changeset/ci-dependencies.md:4]: It's some configuration stuff here no ? 🙄

**Review by jmfrancois — CHANGES_REQUESTED** [review]
After discussion we are agree to switch to a major on @talend/react-cmf-router
We will try to bring settings on it to let teams has the time to plan the migration outside the settings.

**Pull request closed without merging**